STRATEGY.md

```diff
@@ -0,0 +1,22 @@
+# Strategy
+
+## Spark engagement with datasets, collections, and archives
+
+While many datasets are of direct interest only to a few specialists, many others touch on questions that matter deeply to a variety of individuals and interest groups. What can we do to promote critical engagement with this data across a wide spectrum? We’ll need to leverage existing communities and create new ones; collaborate with educational platforms; build new analytic tools and expose existing ones to a wider public. Along the way, we should measure and display the growth of this improved engagement.  
+
+## Demonstrate it is possible for communities to decentralize data they care about
+
+Many of us know that the centralization of data -- scientific data about the natural world, government data about how we are governed, personal data about ourselves -- poses many risks. But we don’t have examples of practical alternatives. Data Together models how we can use decentralized infrastructure to store public data, and to coordinate access, discovery, verification and preservation
+
+## Make it easier to use decentralized data than centralized stores
+
+Our model for data storage will outperform the competition. Provide methods for discovery and use data superior to the existing practices.
+
+## Ingest all the world’s open data onto the Distributed Web
```

Comment:
i noticed that the header of this paragraph says `distributed` and in the body of the paragraph includes both `distributed` and `decentralized`. Perhaps we could use `peer-to-peer`? Thoughts?

Reply:
@ebarry Do you mean to use peer-to-peer as a replacement for those terms, or in addition?

Reply:
I think what's happening here is that the expression "Distributed Web" is a nod to [IPFS](https://ipfs.io)'s tag line.

Backing up for a moment, I _think_ the IPFS team uses that expression as short-hand to communicate the idea of peer-to-peer decentralized and distributed web storage and addressing. However, with due apologies to the excellent people at IFPFS, I'm not sure that outside of IPFS's web pages, the expression is very meaningful. The reason is the following. In the abstract, _distributed_ is about how resources are allocated (specifically, computation and/or storage is distributed across components).  What's weird about the expression "distributed web" is that the web has always been distributed, right from the start, and still is today—sure, there is considerable centralization of resources, but it's certainly not complete. So I worry that the narrative goes too far towards pushing an (IMHO) artificial dichotomy between "centralized web" and "decentralized web".

As for the other terms, my current understanding is that _decentralized_ is usually taken to being about—depending on context—either locality of control (being the opposite of centralized control) or locality of resources (being the opposite of centralized resources), and _peer-to-peer_ is a particular way of implementing something (usually communications) in a distributed and decentralized system.

So coming back to the present document, I think we would be most clear by (1) avoiding the expression "distributed web" unless it's part of a statement like "... using IPFS, the 'distributed web' ...", (2) using _decentralized_ (or _decentralized and distributed_) to describe how data preservation, security, etc., are achieved in DT, and (3) use _peer-to-peer_ as part of the description of how the infrastructure works but not as a replacement for either distributed or decentralized.

Curious about yours, @flyingzumwalt's and other people's views! I know I tend to fixate on technical details and sometimes have trouble perceiving the bigger picture, and maybe this is another instance of that.